4-0-sequencing.md

```diff
@@ -142,11 +166,15 @@ After saving, it is also possible to attach files to the record. Look under the
 
 <a name="trouble"/>
 
-# Troubleshooting
+# 8. Troubleshooting
 
 1. When instruments break down, how do you log failures into the system?
-1. How do you assigning single lane failures (if possible)?
+    * Add a service record as in <a href="#service">7: Adding Service Records to Sequencers</a>
+1. How do you assign single lane failures (if possible)?
+    * It is possible to add a RunQC which allows you to select one or more lanes, but adding one does not change the Orders count for the pool in that lane (ie. it does not "requeue" the pool). At this time, please go to the Edit Pool page for the pool in the lane and add another Order for the number of lanes and sequencing parameters requested.
 1. How you fail flowcells? Or how does MISO detect failed runs?
-1. How do you scan out libraries from inbox to outbox?
+    * If the notification server is aware of the sequencer, then in most cases MISO will automatically detect failed runs if the run has begun sequencing.
+    * If the notification server is not aware of the sequencer or MISO has not detected that the run has failed, then you must select the `Failed` status on the Edit Run page, and enter a completion date.
```

Comment:
Again, I'm not sure we've ever mentioned notification server. It's a low-level detail anyway. Maybe you can revise it to say "MISO is aware of the sequencer"